plugins/commands/compare/src/funTest/kotlin/CompareCommandFunTest.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.commands.compare
+
+import com.github.ajalt.clikt.testing.test
+
+import io.kotest.assertions.withClue
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+class CompareCommandFunTest : WordSpec({
+    "The text diff method" should {
+        "ignore time and environment differences in an analyzer result" {
```

Comment:
Looking at what this tests covers vs. the test assets, I believe the assets contain a lot of unneeded stuff.

I believe it'd be better if the test only had one result hard-coded, then it should make a copy with environment modified and then assert the diff is empty. 

Doing so could re-use an ORT file coded in Kotlin we already have, maybe.

Reply:
Hmm, not sure about this. I deliberately created real-life analyzer results, once from a Jenkins run on Linux, once from a manual run on Windows, to ensure the command works for these, instead of coming up with some artificial results that I'm patching up, with having in mind how the implementations works.

Reply:
So, would you be ok with sticking to the current approach @fviernau?

Reply:
Not really convinced, but the maybe unnecessary maintainance cost I tried to avoid are isolated and limited. So, ok let's make this compromise.